packages/mermaid/src/docs/syntax/gantt.md

```diff
@@ -184,7 +184,7 @@ tickInterval 1day
 The pattern is:
 
 ```javascript
-/^([1-9][0-9]*)(minute|hour|day|week|month)$/;
+/^([1-9][0-9]*)(millisecond|second|minute|hour|day|week|month)$/;
```

Comment:
This change did not automatically modify the `/docs/syntax/gantt.md` (at root) , did you forget to setup pre-commit?

Reply:
Also, this might be great to add `MERMAID_RELEASE_VERSION` to specify from which version `millisecond` and `second` are available, but I also wonder if this feature is important enough to need to be specified

Reply:
> This change did not automatically modify the `/docs/syntax/gantt.md` (at root) , did you forget to setup pre-commit?

Sorry, I am unfamiliar with this setup and don't know what pre-commit is (took a guess and ran `pnpm run pre-commit` and nothing changed).

> Also, this might be great to add MERMAID_RELEASE_VERSION

There is a "Support: v10.3.0+" under that section which could be changed to: "Supported v10.3.0+, `millisecond` and `second` introduced v<MERMAID_RELEASE_VERSION>+."

Reply:
The pre-commit hooks should be setup automatically after pnpm install.

It won't do anything when you run pnpm pre-commit after committing files, as there will not be any changed files to be committed.

You can try running pnpm pre-commit after making some change in the docs file.

Reply:
```diff
diff --git a/packages/mermaid/src/docs/syntax/gantt.md b/packages/mermaid/src/docs/syntax/gantt.md
index 6ddd011f6..a982ab7ad 100644
--- a/packages/mermaid/src/docs/syntax/gantt.md
+++ b/packages/mermaid/src/docs/syntax/gantt.md
@@ -173,7 +173,7 @@ The following formatting strings are supported:
 
 More info in: [https://github.com/d3/d3-time-format/tree/v4.0.0#locale_format](https://github.com/d3/d3-time-format/tree/v4.0.0#locale_format)
 
-### Axis ticks
+### Axis ticks (v10.3.0+)
 
 The default output ticks are auto. You can custom your `tickInterval`, like `1day` or `1week`.
 
@@ -197,7 +197,9 @@ gantt
   weekday monday
 ```
 
-Support: v10.3.0+
+```warning
+millisecond and second support was added in vMERMAID_RELEASE_VERSION
+```
 
 ## Output in compact mode
 
```

You can use the following diff